### PR TITLE
Setup Spring Boot Application

### PR DESCRIPTION
**Pull request by illusi0n**
#1 

Pretty much auto-generated project from https://start.spring.io/

Check package name, artifactId/groupId if changes are needed.

### CODE REVIEW

File: spring/.gitignore [hunk 1]
```diff
@@ -0,0 +1,31 @@
+HELP.md
```
Comment (oskarv) [inline spring/.gitignore]: We have global gitignore https://github.com/oskarv/spring-react/blob/master/.gitignore

File: spring/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
```
Comment (oskarv) [inline spring/pom.xml]: maybe we should use spaces (2 or 4) for tabulation

File: spring/src/main/java/com/springguide/evolvingtodo/EvolvingTodoApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.springguide.evolvingtodo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class EvolvingTodoApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(EvolvingTodoApplication.class, args);
```
Comment (oskarv) [inline spring/src/main/java/com/springguide/evolvingtodo/EvolvingTodoApplication.java]: same here, spaces for tabulation

File: spring/.gitignore (hunk 1 above)
Comment (illusi0n) [reply spring/.gitignore]: Good, we may need to update it once we have separate spring and react sub projects. I think it is a good idea to have separate gitignore for spring and react.

**Comment by illusi0n** [comment]
it seems that the whole project was generated using tabs.

Added these two issues (#7 , #8) based on the feedback.

**Pull request merged**